task/07-yield-tasks.js

```diff
@@ -33,7 +33,23 @@
  *
  */
```

Comment:
Please, provide the link to travis-ci successul build

Reply:
https://travis-ci.org/github/Inquisitor1229/js-assignments/branches